frontend/packages/pipelines-plugin/src/components/taskruns/list-page/TaskRunsRow.tsx

```diff
@@ -66,7 +66,11 @@ const TaskRunsRow: RowFunction<TaskRunKind> = ({ obj, index, key, style, ...prop
       <Timestamp timestamp={obj?.status?.startTime} />
     </TableData>
     <TableData className={tableColumnClasses[7]}>
-      <ResourceKebab actions={Kebab.factory.common} kind={taskRunsReference} resource={obj} />
+      <ResourceKebab
+        actions={props.customData?.hideEditActions ? [Kebab.factory.Delete] : Kebab.factory.common}
```

Comment:
In every instance of `TaskRun`'s kebab action,  we only need to have `Delete TaskRun` option. I don't see a place where we might need to enable edit action. So It could be a simple change `kebab.factory.common` to `kebab.factory.Delete`. @andrewballantyne  any thoughts on this ?

Reply:
I see that you are showing the Edit options on admin side, but we shouldn't show it even there because it redirects to a yaml page that doesn't have `save`  button. 
![image](https://user-images.githubusercontent.com/9964343/118832045-e1c50200-b8dd-11eb-86b1-4b71ec0c78c7.png)